arch/Kconfig

```diff
@@ -476,7 +476,7 @@ if ARCH_KERNEL_STACK
 
 config ARCH_KERNEL_STACKSIZE
 	int "Kernel stack size"
-	default 1568
```

Comment:
It is inappropriate to increase the size of this tuned stack size.  Let's not do that.